requirements/docs.in

```diff
@@ -10,3 +10,4 @@ sphinx-copybutton
 sphinx-design
 pooch
 tqdm
+tof  # For WFM stitching documentation.
```

Comment:
I think this is fine because we are not building the docs as part of the nightly builds.
If we start to do that, then `tof` should also be part of the nightly deps that we checkout @ main.